<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Aalto-Helsinki</title>
    
    <script src="js/jquery-1.6.2.min.js"></script>

    
    <!-- Bootstrap core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    
    <!-- Font Awesome -->
    <link href="css/font-awesome.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/styles-index.css" rel="stylesheet">
    <!--<link href="css/saara.css" rel="stylesheet">
    <link href="css/jami.css" rel="stylesheet">
    <link href="css/emilia.css" rel="stylesheet">-->

    <!-- Just for debugging purposes. Don't actually copy this line! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

    <body class="teambackground">

<div style="height:100%">
    <section class="lakephoto parallax" style="text-align:center;"> 
<h2 class="title">PROJECT</h2>
    </section>
</div>


    <div class="block-wrapper featured-block parallax__layer parallax__layer--base" id="features" style="position: absolute; width:100%; background:white;" >
	    <div class="navbar navbar-fixed-top">
			 <ul class="topnav" style="font-family: robotolight;">
				<li style="float:right"><a href="achievements.html"><b>Achievements</b></a></li>    
				<li style="float:right"><a href="community.html"><b>Community</b></a></li>
				<li style="float:right"><a href="humanpractices.html"><b>Human Practices</b></a></li>
				<li style="float:right"><a href="people.html"><b>People</b></a></li>
				<li style="float:right"><a href="model.html"><b>Modelling</b></a></li>
				<li style="float:right"><a href="lab.html"><b>Laboratory</b></a></li>
				<li style="float:right"><a class="current" href="description.html"><b>Project</b></a></li>
				<li style="float:left; font-family: bebas_neuebold;"><b><a href="index.html" style="margin-left: 10px; font-size:22px; position:absolute;">AALTO-HELSINKI 2016</a></b>
				<!--<li style="float:left"><a href="index.html"><img src='img/logo.png' style="margin-left:3px; width:8%"></img></a></li>-->
			</ul> 
		</div>

    
    <script> $(window).scroll(function() {
    if ($(this).scrollTop() > 610){ // Set position from top to add class
        $('ul.topnav').addClass("shrink");
    } else {
        $('ul.topnav').removeClass("shrink");
    }
    });
    </script>
    
    <div class="block-wrapper-inner" style="padding-top: 15px; position: relative">
		<div class="container" style="width: 75%; font-family: robotolight; color: #4d4d33">
		</br>
			<h1 id="ONE">Introduction</h1>
			<br></br>
			<p style="font-size:19px;" class="justify">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse suscipit arcu libero. Duis vitae cursus elit. Donec enim turpis, aliquam id neque at, faucibus porta odio. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Aliquam et sollicitudin risus. Pellentesque risus magna, accumsan sit amet purus sed, ornare mattis diam. Morbi ac arcu metus. Praesent in massa lectus. Proin fringilla, erat a tempor imperdiet, nunc purus venenatis sem, vel finibus ex lacus a nisi. Morbi nibh augue, ultrices ut lacus id, laoreet aliquam mauris.</p>
		<br></br>
		<div class="ytvideo" align="center">
 				<video width="700" height="500" controls>
  					<source src="video/matti.mp4" type="video/mp4">
  					<source src="video/matti.webm" type="video/webm">
  
					</video></div>
		</div>
		<br></br>
		<div class="sectionjee" style="font-family: robotolight; background-color: #799aa1">
		<h2 style="padding-top: 5px; font-family: im_fell_french_canon_proIt; font-size: 40px;">&ldquo;<i style="font-family: robotolight; font-size: 19px;">Cyanobacteria range in size from 0.5 to 60 micrometers in diameter which represents one of the largest prokaryotic organism.</i>&rdquo;</h2><br></br>
		</div>
	</br>
	<div class="container" style="width: 75%; font-family: robotolight; color: #4d4d33">
		</br>
			<h1 id="TWO">Background</h1>
			<br></br>
			<p style="font-size:19px;" class="justify">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse suscipit arcu libero. Duis vitae cursus elit. Donec enim turpis, aliquam id neque at, faucibus porta odio. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Aliquam et sollicitudin risus. Pellentesque risus magna, accumsan sit amet purus sed, ornare mattis diam. Morbi ac arcu metus. Praesent in massa lectus. Proin fringilla, erat a tempor imperdiet, nunc purus venenatis sem, vel finibus ex lacus a nisi. Morbi nibh augue, ultrices ut lacus id, laoreet aliquam mauris.</p>
		</div>
		<br></br>
		<div class="sectionjee" style="font-family: robotolight; background-color: #799aa1">
		<h2 style="padding-top: 5px; font-family: im_fell_french_canon_proIt; font-size: 40px;">&ldquo;<i style="font-family: robotolight; font-size: 19px;">Cyanobacteria range in size from 0.5 to 60 micrometers in diameter which represents one of the largest prokaryotic organism.</i>&rdquo;</h2><br></br>
		</div>
		<br></br>
		<div class="container" style="width: 75%; font-family: robotolight; color: #4d4d33">
		</br>
			<h1 id="THREE">Detection</h1>
			</br>
			<h3>Background</h3>
			</br>
			<p style="font-size:19px;" class="justify">When studying the toxicity of the cyanobacterial toxin microcystin-LR in higher eukaryotic cells, we found out that it causes oxidative stress in them. This is due to inhibition of protein phosphatases 1 and 2A (PP1 and PP2A) which then leads to hyperphosphorylation of target proteins. This excessive phosphorylation in turn increases the formation of reactive oxygen species (ROS), resulting in a oxidative stress response. (Campos and Vasconcelos, 2010) This is presented in the figure 1. Reactive oxygen species have many effects in the cell. For example, catalase production is significantly increased in the presence of ROS. Catalase is an enzyme found in nearly all living organisms that catalyses the decomposition hydrogen peroxide. It is one of the most important enzymes in protecting the cell from reactive oxygen species meaning that its production is activated when there is oxidative stress. We found out that similar toxicity effects can be observed in some lower eukaryotes, such as the yeast <i>Saccharomyces cerevisiae</i>. (Valério et al., 2014, Valério et al., 2016) Realizing this made us think of how we could use MC-LR's natural toxicity effects to sense the toxin concentrations.
			</p>
			<figure style="float: right; text-align:center;">
			<img style="width:50%; height; 50%" src='img/toxicity.png'><br><figcaption style="padding-bottom: 20px">Figure 1. Effects of microcystin-LR in eukaryotic cells.</figcaption>
			</figure>
			<br></br>
			<h3>Stress Promoters</h3>
			</br>
			<p class="justify" style="font-size:19px">Various articles (Estruch, 2000, He et al., 2005) indicated that oxidative stress increases the activity of certain transcription factors in <i>S. cerevisiae</i> cells. Two transcription factors activated by oxidative stress are  Skn7p and Yap1p. These transcription factors have specific binding sites in the promoter areas of genes that respond to oxidative stress. Skn7p binds to OSRE (oxidative stress response element) and Yap1p binds to YRE (Yap1p response element). Some of the best characterized promoter regions that these transcription factors bind to are those for the genes TSA1 (thioredoxin peroxidase) and CCP1 (mitochondrial cytochrome-c peroxidase) (He et al. 2005). The gene that was reported to have highest response against microcystin was CTT1 (cytosolic catalase T) (Valério et al., 2014). We chose to look into these three promoters more deeply, and this led us to the idea of creating a sensor by fusing one of these promoter areas with fluorescent protein production and implementing this in <i>S. cerevisiae</i>.
			</p>
			</br>
			<p class="justify" style="font-size:19px">This sensor would produce a fluorescent signal when the activation of the transcription factors Skn7p and Yap1p has been induced – in other words, when the <i>S. cerevisiae cells encounter stress</i>. As microcystin is a rather effective stress-generating factor, and as most sources of oxidative stress (such as xenobiotics and heavy metals) are rare in nature (Farrugia et al., 2012), we can assume that it is the main stress producing factor found in Finnish lakes. This leads to our hypothesis: fluorescent colour produced by our engineered <i>S. cerevisiae</i> is proportional to microcystin concentration in measured water samples.</p>
			</br>
			<h3>Transporter</h3>
			</br>
			<p class="justify" style="font-size:19px">An important prerequisite for the function of the stress promoters is that the toxin must be imported into the cell. Microcystin-LR doesn’t pass the cell membrane without the help of specific transporter proteins. In human liver cells, the membrane bound OATP transporter (organic anion transport peptide) was identified to imports MCs into the cell. OATPs belong to the superfamily of solute carriers and mediate the uptake of endogenous compounds such as hormones or bile salts but also a variety of xenobiotics (Faltermann et al., 2016). In yeast, the QDR2 transporter has a similar function. It has a broad substrate specificity and can transport many mono- and divalent cations as well as a variety or drugs.  According to Valerio et al., the QDR2 transporter from the yeast strain VL3 is the most similar to human OATP out of all  <i>S. cerevisiae</i> strains.We also did a BLAST search to confirm this data and come to the conclusion that QDR2 transporter from VL3 corresponded best to the human OATP transporter out of all yeast transporters. Transporters from all other common <i>S. cerevisiae</i> strains differ significantly from these.</p>
			</br>
			<p class="justify" style="font-size:19px">It has been confirmed that the QDR2 transporter from VL3 can import microcystin (Valério et al., 2014). We reasoned that by replacing the original QDR2 transporter of a <i>S. cerevisiae</i> lab strain (W303a/SS238) with the VL3 variant of the QDR2 transporter we could enable toxin import in a new strain. The reason for not wanting to work with VL3 was that it’s used and optimized for wine production, not for extensive gene engineering. Another reason for not choosing to engineer VL3 was that we would have had to do auxotrophic mutations for it first as such kind of this strain was not available. However, we planned to use VL3 as a positive control in our experiments to compare how changing the transporter of a lab strain effects observed toxicity effects. This way, we hope to see whether toxicity behaviour corresponding to VL3 can be observed.
			</p>
			</br>
			<h3>Final Product</h3>
			<!--<figure style="float: right; text-align:center; padding-left:20px;">
			<img style="width:80%;" src='img/toxicity.png'><br><figcaption>Figure 1.</figcaption>-->
			</br>
			<p class="justify" style="font-size:19px">For the final product, we will need to have both the VL3 transporter and the stress promoter in yeast cells.  The colour of two lake water samples, one supplemented with a defined quantity of engineered <i>S. cerevisiae</i> cells and one with unmodified yeast, can then be compared, and the toxin concentration can be calculated easily from a standard curve utilizing known toxin concentrations.
			<br></br>In the final product, we would have a test device containing the measurement and control yeasts. The user could add a small amount of sample water to be measured and analyse it with a mobile app that tells about the toxicity based on the colour difference between the sample and the control. The idea is explained shortly in figure 2. </p>
			</br>
			<figure style="float: right; text-align:center;">
			<img style="width:70%;" src='img/detection1.png'><figcaption style="padding-bottom: 30px">Figure 2. Simplified detection principle.</figcaption>
			</figure>
			<br></br>
			<h2>Design</h2>
			<h3>Promoters</h3>
		</br>
			<p class="justify" style="font-size:19px">We chose three whose transcription is regulated greatly by oxidative stress or which promoters are well characterized. These genes were TSA1 (Saccharomyces genome database ID: S000004490), CTT1 (SGD ID: S000003320) and CCP1 (SGD ID: S000001774). We decided to incorporate the promoters of these genes directly into our stress promoter constructs. Binding sites for transcription factors of our interest are identified the region of 1-300 bp before the start codon, but the length of the whole promoter area has not been defined. As promoter areas can be up to 1000 bp long, we decided to use upstream sequences longer than 300 bp. For TSA1 we decided take promoter region of 260 base pairs, as that was the longest we could take so that it didn’t overlap with the previous gene in yeast genome. In the case of CCP1, we decided to have a promoter region of 756 bp, as that was the same that had been used and proved to be working in our reference article (He et al., 2005). Unlike for CCP1 and TSA1, we didn’t find as clear an explanation for the length of CTT1. We only found that the reference article had done its experiment with 479 upstream sequence so we decided to be sure that it was long enough and took 924 bp. 
			<br></br>
			For our synthesized gene constructs, these promoters were then directly followed by the coding region of a yellow fluorescent protein. This construct was designed to replace the original promoter and multiple cloning site of our yeast plasmid. This way we could make YFP production dependent on oxidative stress. Venus yellow fluorescent protein was chosen as the visual reporter as it was supposed to be bright, stable in a variety of conditions (Shaner et al., 2015) and the sequence of codon optimized version for <i>S. cerevisiae</i> was available. Besides promoter + venus constructs, we decided to expressed Venus under control of one constitutive and one inducible promoter to get more reliable information about expression of Venus in yeast, and to be able to study the effect of different conditions on its expression under known promoters. The used promoters were the galactose inducible GAL1 promoter and the strong, constitutive GPD1 promoter. These constructs will serve as a control in our stress promoter tests.
			<br></br>
			The constructs were designed so that they were compatible with both Gibson assembly and yeast recombination cloning. This means that a sequence of 30 bp that is homologous with the desired cloning location from the plasmid backbone was added to each end of the promoter + Venus construct. As a plasmid backbone, we used pRS415-GAL/GPD (pRS415 plasmid with either GPD1 or GAL1 promoters), which was already available in our lab, and suitable for protein expression in <i>S. cerevisiae.</i> A simplified picture of all constructs is presented in figure 3.  Inserts containing homologous sites for ligation, designed promoter, and venus, were ordered from IDT.
			</br></p>
			<figure style="float: right; text-align:center;">
			<img style="width:100%;" src='img/promoters2.png'><figcaption style="padding-bottom: 30px">Figure 3. Simplified diagrams of our detection plasmids.</figcaption>
			</figure>
			<br></br>
			<p class="justify" style="font-size:19px">Our designed inserts are not compatible with Biobrick assembly standard 10. The reason for this was that there were no plasmid backbones or hardly any other parts designed for <i>S. cerevisiae</i> in stock in the iGEM registry. Additionally, the characterization of the parts were lacking. The plasmid backbone we eventually chose was pRS415-GAL, as it was easily available to us, suitable for our purpose, and there was experience that it had previously worked well in <i>S. cerevisiae</i>. Our plan was to modify the best of our designed inserts to be compatible with Biobrick standard 10 (to have defined prefix and suffix) after testing and analyzing them. 
			<br></br>
			Our plan to test the functionality of stress promoters is to use hydrogen peroxide to induce stress in yeast cells containing stress promoter plasmids, and measure the differences in YFP fluorescence intensities. From these results, we hope to obtain a linear relation between inducing hydrogen peroxide concentration and YFP intensity. These tests cannot be done with microcystin-LR, since it isn’t imported into the cell without the VL3 transporter, and thus doesn’t cause a strong enough response. All the same, as hydrogen peroxide is the main reactive oxygen species produced in stress caused by microcystin-LR in yeast cells, we planned to use this compound directly in our primary experiments. After proving that our sensor works with hydrogen peroxide, and finishing transporter implementation, we can then transform our stress promoter into cells with the VL3 transporter already integrated into their genome. This way, we could test our promoters with microcystin.
			<br></br></p>
			<h3>Modelling</h3>
			<br></br>
			<p class="justify" style="font-size:19px">We are going study our detection mechanism mathematically in order to obtain better understanding of it. Our plan is to create simplified molecular and mathematical models of our detection mechanism which will describe  how the transcription factors are affected and how they affect the regulation of our promoters under oxidative stress. Then, we will simulate these models with COPASI modelling software. We hope to validate our model with experimental results for each construct. In addition, we hope to determine theoretical detection thresholds and measurement accuracy estimates, as well as provide suggestions for optimal promoter design for future iterations of the project.
			 <br></br>
			As modeling goes into great detail on subjects not as tightly related to the biological design of the project, it is presented in more detail in it’s <a href="model.html" style="color:#799aa1"><b>own page</b></a>.
			<br></br></p>
			<h3>Transporter</h3>
			<br></br>
			<p class="justify" style="font-size:19px">The QDR2 transporter sequences of VL3 strain were available online, but there were two nucleotides in the sequence that were not determined. The first one (nucleotide 475) was marked with M, meaning that it could be either adenine (A) or cytosine (C), and the other one (nucleotide 1126) with Y, meaning that it was either tyrosine (T) or cytosine (C). For the first nucleotide, the different bases would result in a changed amino acid at position 159 -  either a leucine or isoleucine. We decided to test both transporter variants. For the second unclear base, replacing Y with T would have produced a premature stop codon at the place of amino acid 376. Because of this, we chose C, which resulted in having glutamine as amino acid 376. After this reasoning we had two different transporter variants. The following figure 4 presents the possible options we had.
			<br></br></p>
			<figure style="float: right; text-align:center;">
			<img style="width:70%;" src='img/detection2.png'><figcaption style="padding-bottom: 30px">Figure 4. There were two unknown bases in the sequence of QDR2 transporter of VL3. Our plan was to test all options, except the one leading to a premature end of transcription.</figcaption>
			</figure>
			<br></br>
			<p class="justify" style="font-size:19px">We planned to replace the transporter directly in our <i>S. cerevisiae</i> lab strain’s genome with homologous recombination. To be able to detect the correct expression of the transporter replacement in the eventuality that functional testing with response to MC wouldn’t produce clear enough results, we also designed constructs with strep-tag markers on both termini and a construct with a mCherry red fluorescent protein tag on the C-terminus. All designed QDR2 variants are presented in figure 5.
			</p>
			<figure style="float: right; text-align:center;">
			<img style="width:20%;" src='img/wikiparts1.png'><figcaption style="padding-bottom: 30px">Figure 5. All designed variants of the QDR2 transporter from VL3. </figcaption>
			</figure>
			<br></br>
			<p class="justify" style="font-size:19px">All inserts were designed to be amplified in the pUG6 plasmid backbone, which is commonly used to generate deletion cassettes to be used in yeast homologous recombination (Güldener et al., 1996). The pUG6 backbone contains ampicillin resistance for amplification in <i>E. coli</i>, and a KanMX resistance cassette for antibiotic selection in yeast, flanked by loxP sites to enable recovery of the selection marker after selection. The inserts were designed to be compatible with either Gibson assembly or yeast recombination with a linearized pUG6 plasmid backbone. The disruption cassette for genome integration can then be generated from the assembled transporter-pUG6 plasmid via PCR. The PCR is designed to generate a linear fragment containing both the transporter, the KanMX selection marker, and 50 bp long regions homologous to the intended site of recombination in the genome. (During this PCR, 50 bp long homology will be added to the 3’ end to enable genome integration. 5’ homology is designed already in the ordered insert.) This linear cassette can be then transformed into our lab strain, where it will integrate into the location of original transporter. The planned integration is illustrated in figure 6.
			<br></br></p>
			<figure style="float: right; text-align:center;">
			<img style="width:70%;" src='img/kaavio.png'><figcaption style="padding-bottom: 30px">Figure 6. Integration of the VL3 QDR2 transporter into the genome of SS328.</figcaption>
			</figure>
			<br></br>
			<p class="justify" style="font-size:19px">For the same reason as in the case of stress promoters, these inserts aren’t compatible with Biobrick assembly standard 10.
			
			
		</div>
		<br></br>
	
		<div class="sectionjee" style="font-family: robotolight; background-color: #799aa1">
		<h2 style="padding-top: 5px; font-family: im_fell_french_canon_proIt; font-size: 40px;">&ldquo;<i style="font-family: robotolight; font-size: 19px;">Cyanobacteria range in size from 0.5 to 60 micrometers in diameter which represents one of the largest prokaryotic organism.</i>&rdquo;</h2><br></br>
		</div>
		<div class="container" style="width: 75%; font-family: robotolight; color: #4d4d33">
		<br></br>	
				
		<h1 id="FOUR">Degradation</h1>
		</br>
		<h3>Background</h3>
		</br>
		<p class="justify" style="font-size:19px">The second part of our project, the degradation of the cyanobacterial toxin microcystin (MC), is also engineered in the yeast <i>Saccharomyces cerevisiae</i>. We want to produce an MC-degrading enzyme called microcystinase (MlrA) which is found naturally in some bacterial species. Our goal is to produce an enzyme capsule which can be added into MC-containing water where it degrades the toxins and makes the water safe to use.</p>
		</br>
		<p class="justify" style="font-size:19px">There is a gene cluster which has been identified to code for enzymes which degrade microcystins. The genes mlrA, mlrB, mlrC and mlrD code for enzymes MlrA, MlrB, MlrC and MlrD respectively. They all have a role in the degradation process with different functions. MlrA, also known as microcystinase, acts first by hydrolysing the bond between microcystin’s amino acids Adda and arginine. This linearizes the toxin, after which MlrB (a serine protease), and MlrC (a metalloprotease) are able to degrade it further to individual amino acids, which are then used in the cell’s metabolism. MlrD’s function is not fully understood at this point, but it may have a role in importing the toxins. (1)</p>
		</br>
		<p class="justify" style="font-size:19px">Our focus will be on the degradation of MC-LR because it is the most common cyanotoxin and also because the degradation has been characterized for this toxin. The linearization, performed by MlrA, renders the toxin 160 less toxic (2). This should be enough to make for example a bucket of water safe to use as sauna water (check some calculation about this in results somewhere).
		</p>
		</br>
		<p class="justify" style="font-size:19px">Since the cyanobacteria release the toxins only as they lyse, we have to take also that into consideration. Based on earlier work by Peking 2014 iGEM team, hen egg lysozyme is good to lyse the tough cell walls of cyanobacteria WE CAN’T USE LYSOZYME. Our final product will contain lysozyme and microcystinase. The lysozyme would first lyse the cyanobacteria and release all the possible MC, and the microcystinase which would then linearize the toxins.</p>
		</br>
		<p class="justify" style="font-size:19px">PICTURE</p>
		</br>
		<p class="justify" style="font-size:19px">MlrA is a putative membrane protein. It has eight membrane spanning helices and according to our homologous structure model it forms a channel, in middle of which the active site resides.  Although MlrA seems to be a membrane protein, there has been studies, where at least some of it is located in the cytosol (3).</p>
		</br>
		<p class="justify" style="font-size:19px">PICTURE</p>
		</br>
		<p class="justify" style="font-size:19px">There has been a previous iGEM project about degradation of microcystin (Ranger amongst Enemies, Peking 2014). Their device was engineered in E. coli, which expressed MlrA as a part of their project. The MlrA sequence used was from Sphingomonas, but the exact species was not mentioned. Their BioBrick of the enzyme differed in one amino acid compared to the one we found in GenBank and that we have used in our work.</p>
		</br>
		<h3>Design</h3>
		</br>
		<p class="justify" style="font-size:19px">We have three different variants of the enzyme for production in <i>S. cerevisiae</i> as well as a positive control for production in <i>E. coli</i>, since it has been already established that heterologous production of MlrA results in active enzyme (3). </p>
		</br>
		<p class="justify" style="font-size:19px">We will use the MlrA sequence from <i>Sphingomonas sp. ACM-396</i> (LINK TO GENBANK), as it has been characterized to degrade MC-LR, and because apparently it was the one used in Peking 2014 team’s work. To make sure that we get the best possible protein yields from expression in yeast and <i>E. coli</i>, codon optimization was required.</p>
		</br>
		<p class="justify" style="font-size:19px">In mass production the most efficient way to produce proteins is having them secreted into the growing media. This poses a problem as our enzyme is putatively a membrane protein. In the effort of having it secreted, we are using a yeast-specific secretion tag, mating factor α. Mating factor α is also used to localize membrane proteins into the plasma membrane. We will produce four different versions of the enzyme: MlrAY, MlrAYAlpha, MlrAY3Alpha and MlrAE. </p>
			<ul style="font-size:19px">
				<li>MlrAY, containing no mating factor α, should be soluble in the cytosol.</li> 
				<li>MlrAYAlpha has one secretion tag, which should localize the protein into the plasma membrane. </li>
				<li>MlrAY3Alpha has a secretion tag with a triple propeptide part, and so is the most likely of the three proteins to be secreted, if at all. </li>
				<li>MlrAE, used as a positive control by production of the enzyme in E. coli. It has been demonstrated earlier that the enzyme is active in this production system. </li>
			</ul>
		</p>
		<p class="justify" style="font-size:19px">For the yeast backbone we are using pRS415-GAL whereas pSVA1 is used as the bacterial backbone. Both are available to us at our lab.</p>
		</br>
		<p class="justify" style="font-size:19px">For purification of the protein, we are putting a 6xHis-tag into the C-terminal of the protein. This should allow for a simple purification using Immobilized Metal Affinity Chromatography (IMAC). If the enzyme localizes in membranes with the secretion tag(s), the best way to purify it while making sure it is still active is to purify the membranes as a whole. This means that the enzyme would be in its natural environment in the purified form.</p>
		</br>
		<p class="justify" style="font-size:19px">In the design of the degradation part of our project we won’t be using BioBricks simply because there aren’t any appropriate ones available/well characterized. Our designs weren’t compatible with the BioBrick standard, and we didn’t want to compromise them in order to have for example a ready BioBrick backbone.</p>
		</br>
		<p class="justify" style="font-size:19px">PICTURE</p>
		
		
		</div>

		<br></br>
	
		<div class="sectionjee" style="font-family: robotolight; background-color: #799aa1">
		<h2 style="padding-top: 5px; font-family: im_fell_french_canon_proIt; font-size: 40px;">&ldquo;<i style="font-family: robotolight; font-size: 19px;">Cyanobacteria range in size from 0.5 to 60 micrometers in diameter which represents one of the largest prokaryotic organism.</i>&rdquo;</h2><br></br>
		</div>
		<br></br>
			<div class="container" style="width: 75%; font-family: robotolight; color: #4d4d33">
			<h2>References</h2>
				<br></br>
				<p style="font-size:19px;" class="justify">Chorus I, Bartram J, eds., 1999. Toxic Cyanobacteria in Water: A Guide to Their Public Health Consequences, Monitoring, and Management. London: E & FN Spon.
				<br></br>
				Dziga, D., Wladyka, B., Zielińska, G., Meriluoto, J., Wasylewski, M., 2012. Heterologous expression and characterisation of microcystinase (2012) Toxicon, 59 (5), pp. 578-586.
				<br></br>
				Edwards C, Graham D, Fowler N, Lawton LA., 2008. Biodegradation of microcystins and nodularin in freshwaters. Chemosphere, 73(8), pp. 1315-1321.
				<br></br>
				Gasch, A.P., Spellman. P.T., Kao, C.M., Carmel-Harel, O., Eisen, M.B., Storz, G., Botstein, D., Brown, P.O., 2000. Genomic expression programs in the response of yeast cells to environmental changes. Mol Biol Cell., 11, pp. 4241–4257.
				<br></br>
				Iwase T, Tajima A, Sugimoto S, Okuda KI, Hironaka I, Kamata Y, Takada K, Mizunoe Y., 2013. A simple assay for measuring catalase activity: a visual approach. Sci. Rep. 3, pp. 3081.
				<br></br>
				Valério, E., Vilares, A., Campos, A., Pereira, P. and Vasconcelos, V., 2014. Effects of microcystin-LR on Saccharomyces cerevisiae growth, oxidative stress and apoptosis. Toxicon, 90, pp.191-198.
				<br></br>
				Veal, E.A., Ross, S.J., Malakasi, P., Peacock, E. and Morgan, B.A., 2003. Ybp1 is required for the hydrogen peroxide-induced oxidation of the Yap1 transcription factor. Journal of Biological Chemistry, 278(33), pp.30896-30904.
				<br></br>
				WHO, 2003. Cyanobacterial toxins: Microcystin-LR in drinking-water. Background document for preparation of WHO Guidelines for drinking-water quality. Geneva, World Health Organization (WHO/SDE/WSH/03.04/57).
				<br></br>
				</p>
		</div>

		
			
	</div>

		<script>

$(window).scroll(function(){
	$(".title").css("opacity", 1 - $(window).scrollTop() / 500);
});

</script>

	

    
    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <!--script src="./Tinyone - HTML_files/jquery.min.js"></script>
    <script src="./Tinyone - HTML_files/bootstrap.min.js"></script>
    <script src="./Tinyone - HTML_files/offcanvas.js"></script>
    <script src="./Tinyone - HTML_files/function.js"></script-->
  

</body></html>